compiler/hash-typecheck/src/exhaustiveness/constant.rs

```diff
@@ -29,8 +29,8 @@ impl Constant {
     /// Function to convert a [BigInt] into a [Constant]. The only
     /// constraint is that it can fit into a [u128], otherwise the
     /// function will currently panic.
-    pub fn from_int(int: BigInt, kind: IntTy, ty: TermId) -> Self {
-        let size = kind.size().unwrap_or_else(|| int.bits());
+    pub fn from_int(int: BigInt, kind: IntTy, ty: TermId, ptr_width: usize) -> Self {
+        let size = kind.size(ptr_width).unwrap_or_else(|| int.bits());
```

Comment:
int.bytes()?

Reply:
Although, I think it is easier to multiply the returned integer size by 8 to convert it into a bit count instead of using `bigint` to get the number of bytes since it seems they don't have an API for providing the size of the number in bytes.